src/middleware/site.js

```diff
@@ -8,7 +8,7 @@ module.exports = function( req, res, next ) {
 	let siteId;
 
 	// deze paden mogen dit overslaan
-	if (req.path.match('^(/doc|/dev|/accepteer-cookies|/$)')) return next();
+	if (req.path.match('^(/doc|/dev|/accepteer-cookies|/api/area|/$)')) return next();
```

Comment:
add todo, to refactor the site middleware (the site middleware should not be global for every path)